include/mixer.h

```diff
@@ -54,8 +55,19 @@ enum MixerModes {
 
 // A simple stereo audio frame
 struct AudioFrame {
-	float left = 0;
+	float left  = 0;
 	float right = 0;
```

Comment:
Suggest literal intializers that match the type, so `0.0f` here.

(With narrowing enabled, (I think) it would warn about the integer losing precision on the assignment.)

Reply:
Ah yes.